src/html.htl

```diff
@@ -1,9 +1,23 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <title>${content.title}</title>
-    <meta data-sly-test.hash=${content.data.sourceHash} name="x-source-hash" content="${hash}">
-    <esi:include src="/head.html" onerror="continue"/>
+  <title>${content.title}</title>
+  <meta data-sly-test.hash=${content.data.sourceHash} name="x-source-hash" content="${hash}"/>
+  <link data-sly-test="${content.meta.url}" rel="canonical" href="${content.meta.url}"/>
+  <meta data-sly-test.desc="${content.meta.description}" name="description" content="${content.meta.description}"/>
+  <meta data-sly-test="${content.title}" property="og:title" content="${content.title}"/>
+  <meta data-sly-test="${content.meta.description}" property="og:description" content="${content.meta.description}"/>
+  <meta data-sly-test="${content.meta.url}" property="og:url" content="${content.meta.url}"/>
+  <meta data-sly-test="${content.meta.imageUrl}" property="og:image" content="${content.meta.imageUrl}"/>
+  <meta data-sly-test="${content.meta.imageUrl}" property="og:image:secure_url" content="${content.meta.imageUrl}"/>
+  <meta data-sly-test="${content.meta.modifiedTime}" property="og:updated_time" content="${content.meta.modifiedTime}"/>
+  <sly data-sly-test="${content.meta.tags}" data-sly-list.tag="${content.meta.tags}">
+    <meta property="article:tag" content="${tag}"/>
+  </sly>
+  <meta data-sly-test="${content.meta.section}" property="article:section" content="${section}"/>
+  <meta data-sly-test="${content.meta.published_time}" property="article:published_time" content="${published}"/>
+  <meta data-sly-test="${content.meta.modified_time}" property="article:modified_time" content="${modified}"/>
```

Comment:
Where is `modified` coming from ? Why not `content.meta.modifiedTime` like above ?